src/main/scala/scala/tools/refactoring/implementations/OrganizeImports.scala

```diff
@@ -407,7 +407,8 @@ abstract class OrganizeImports extends MultiStageRefactoring with TreeFactory
   class RefactoringParameters(
     val importsToAdd: List[(String, String)] = Nil,
     val options: List[Participant] = DefaultOptions,
-    val deps: Dependencies.Value = Dependencies.RemoveUnneeded)
+    val deps: Dependencies.Value = Dependencies.RemoveUnneeded,
+    val organizeLocalImports: Boolean = true)
```

Comment:
@wpopielarski if you enable this by default (against my suggestion from above - but I don't insist), please make sure that you don't forget about making this configurable in the IDE. This will also help with debugging, as we can then see very quickly if bugs go away as we enable/disable this.